src/transforms/bin.js

```diff
@@ -311,20 +313,11 @@ export function maybeThresholds(thresholds, interval, defaultThresholds = thresh
       case "auto":
         return thresholdAuto;
     }
-    const interval = maybeInterval(thresholds);
-    if (interval !== undefined) return interval;
-    throw new Error(`invalid thresholds: ${thresholds}`);
+    return maybeUtcInterval(thresholds);
```

Comment:
At this point, we’ve already checked that `typeof thresholds === "string"`, so the other code paths should not be possible.